verifact/forum/models.py

```diff
@@ -10,3 +11,22 @@ class Question(models.Model):
 
     def __str__(self):
         return self.text
+
+
+class Answer(models.Model):
+    class AnswerTypes(models.IntegerChoices):
+        TRUE = 1, _("True")
+        FALSE = 2, _("False")
+        OTHER = 3, _("Other")
```

Comment:
I guess we never came to a final decision about this, but `Other` strikes me as something that might confuse developers down the road. Let's have a brief discussion about this in the tech channel to get consensus.

Reply:
changing both enum and display text to "Neither" for now as mentioned on whatsapp